src/app/ReadHandler.h

```diff
@@ -429,6 +428,7 @@ class ReadHandler : public Messaging::ExchangeDelegate
     //
     friend class chip::app::reporting::Engine;
     friend class chip::app::InteractionModelEngine;
+    friend class chip::app::SubscriptionResumptionHelper;
```

Comment:
Why does this need to be a friend class? Does it make more sense to move what is needed from private to public?

Reply:
Move the private function to public now